bentoml/archive/templates.py

```diff
@@ -150,3 +149,57 @@ def load():
 
 __all__ = ['__version__', '{service_name}', 'load']
 """
+
+BENTO_SERVICE_DOCKERFILE_CLIPPER_TEMPLATE = """\
+FROM clipper/python36-closure-container:0.3
```

Comment:
add a note on why we use python36 image, even for BentoArchive built in python27